docs/src/installation.md

```diff
@@ -1,85 +1,128 @@
-Installation Guide
-==================
+# Installation Guide
 
 JuMP is a package for [Julia](https://julialang.org). To use JuMP, first
-[download and install](https://julialang.org/downloads/) Julia or open up
-a remote notebook at [JuliaBox](https://www.juliabox.com/) or similar services.
+[download and install](https://julialang.org/downloads/) Julia.
 
-This version of JuMP is compatible with Julia 1.0 and later.
+!!! note
+    This version of JuMP is compatible with Julia 1.0 and later.
 
-From Julia, JuMP is installed by using the built-in package manager:
+From Julia, JuMP is installed using the built-in package manager:
 ```julia
 import Pkg
 Pkg.add("JuMP")
 ```
+## Installing a solver
 
-Getting Solvers
----------------
+JuMP depends on solvers to solve optimization problems, and you will need to
+install one before you can solve problems with JuMP. The table below lists the
+currently available solvers.
 
-JuMP depends on solvers to solve optimization problems. Most solvers are not
-written in Julia, and some require commercial licenses to use, so installation
-is often more complex. We list below the currently available solvers.
+Common choices of solvers are [Clp.jl](https://github.com/jump-dev/Clp.jl) for
```

Comment:
I'd put solver recommendations aside for a separate PR. JuMP hasn't officially done this before, so it needs some discussion.